### PR TITLE
Describe use case for document.languages

### LINKED ISSUE #930: Publishing a document available in multiple languages
Afghanistan has two official languages, and may therefore want to publish a document that is available in both languages at the same URL e.g. contracts. 

However, `document.language` is a string rather than an array of strings. 

For options now, @duncandewhurst suggested these alternatives:
- If one language is the primary language in any sense, then put this in the `.language` field and explain in the publication policy and/or `.description` field that the document is also provided in the secondary language.
- Add two entries to the `.documents` array, one for each language.

For the future, @jpmckinney suggested that one approach to avoid having two entries for the same document is to add `.languages` (array) and deprecate `.language` (string).

See also: CRM-5197.

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #930

### CODE REVIEW

**Review by mrshll1001 — COMMENTED** [review]
The difference between documents containing multiple languages and a document published in multiple languages may not be obvious at first read. I, a native English speaker, had to read the paragraph a few times to understand the difference.

I suggest modifying the second sentence re the "documents published in multiple languages" to be clear that we're talking about a document which has several different published versions rather than a single document containing multiple languages.

If I've misunderstood this, then I think this adds weight to the need to re-phrase this slightly.

**Review by mrshll1001 — APPROVED** [review]
Looks great, very clear!

File: docs/history/changelog.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
     * `contracts.maximumValue`
     * `contracts.estimatedValue`
   * [#1372](https://github.com/open-contracting/standard/pull/1372) `Address.country`
-  * [#1380](https://github.com/open-contracting/standard/pull/1380) `Document.languages`
+  * [#1380](https://github.com/open-contracting/standard/pull/1380), [#1432](https://github.com/open-contracting/standard/pull/1432) `Document.languages`
```
Comment (jpmckinney) [inline docs/history/changelog.md]: No comma between PR/issue numbers.

```suggestion
  * [#1380](https://github.com/open-contracting/standard/pull/1380) [#1432](https://github.com/open-contracting/standard/pull/1432) `Document.languages`
```

File: docs/schema/reference.md [hunk 2]
```diff
@@ -69,16 +69,16 @@ The majority of OCDS data is held within a release structure. One or more releas
 
 * [package](#package-metadata)
   * [release](#release)
-    * [parties](#parties)
```
Comment (jpmckinney) [inline docs/schema/reference.md:72]: Please re-do your changes from a new, clean branch, but without these whitespace changes.

File: docs/schema/reference.md [hunk 3]
```diff
@@ -317,6 +317,8 @@ Documents should be published at their own stable URLs, accessible for free and
 
 OCDS allows summarizing information in the document's `description` field. Providing clear summaries is a good practice, as it allows applications to display this information in a user-interface and thus enables users to read key facts without having to search through the whole document.
 
+If a document contains multiple languages, use the `languages` field to list the languages used in the document. If multiple versions of a document are published, each in a different language, add a separate `Document` object for each version of the document.
```
Comment (jpmckinney) [inline docs/schema/reference.md]:
```suggestion
If a document contains multiple languages, use the `languages` field to list the languages used in the document. If there are multiple versions of a document, each in a different language, add a separate `Document` object for each version of the document.
```

File: docs/schema/reference.md (hunk 2 above)
Comment (duncandewhurst) [reply docs/schema/reference.md:72]: https://github.com/open-contracting/standard/pull/1440

**Pull request closed without merging**